solana-programs/audius_eth_registry/tests/tests.rs

```diff
@@ -54,6 +57,69 @@ async fn setup() -> (BanksClient, Keypair, Hash, Keypair, Keypair) {
     )
 }
 
+pub fn new_secp256k1_instruction_2_0(
```

Comment:
why are we copying this one out rather than using the native solana function?

Reply:
doesn't exist